google/utils.go

```diff
@@ -318,6 +318,14 @@ func convertStringSet(set *schema.Set) []string {
 	return s
 }
 
+func convertStringMap(v map[string]interface{}) map[string]string {
```

Comment:
expandStringMap does the same as this i believe

Reply:
Not quite. `expandStringMap` works with a `schema.ResourceData`. This one works with a `map[string]interface{}`.

Reply:
Can you rewrite one in terms of the other? `expandStringMap` can defer to this method

Reply:
Done